calc/src/test/calc.test.ts

```diff
@@ -956,6 +956,21 @@ describe('calc', () => {
           defender.teraType = 'Normal';
           expect(calc(cc)).toEqual(se);
         });
+        test('Quark Drive should not take into account boosted stats', () => {
```

Comment:
We should be testing both

Reply:
Both quark and Proto? In that case I'll change the implementation from Booster Energy to the respective field effects, which should make the tests more powerful.